src/sonic-config-engine/tests/test_multinpu_cfggen.py

```diff
@@ -304,6 +304,7 @@ def test_device_asic_metadata(self):
                 self.assertEqual(output['localhost']['sub_role'], 'FrontEnd')
             else:
                 self.assertEqual(output['localhost']['sub_role'], 'BackEnd')
+            self.assertEqual(output['localhost']['deployment_id'], "1")
```

Comment:
If there's no DeploymentId, what's the expected output?

Reply:
the behavior will be same as your PR: https://github.com/Azure/sonic-buildimage/pull/10154. It will be none and not get populated in result dictionary